src/functions-and-arrays.js

```diff
@@ -29,9 +83,34 @@ const wordsUnique = [
   'bring'
 ];
 
+const uniquifyArray = array =>{
+  var uniqueArray = [];
+  if(!array.length){
+    return null;
+  } for(i=0; i < array.length; i++){
+    if(uniqueArray.indexOf(array[i]) === -1){
+        uniqueArray.push(array[i]);
+    }
+  }
+    return uniqueArray;
+}
+
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
+const doesWordExist = (words, value) => {
```

Comment:
All tests are running successfully, great! Perhaps there is a way to code this functionality in a more minimal way? Please check out the following Stackoverflow thread. https://stackoverflow.com/questions/6116474/how-to-find-if-an-array-contains-a-specific-string-in-javascript-jquery